xtask/src/binutil.rs

```diff
@@ -33,7 +33,14 @@ impl LlvmTools {
 				return Ok(Self { bin });
 			}
 		}
-		unreachable!()
+		eprintln!(
+			"Error: Could not find the required LLVM tool '{}'. Searched in: {:?}",
+			example_exe, rustlib
+		);
+		Err(io::Error::new(
+			io::ErrorKind::NotFound,
+			"Required LLVM tool not found",
+		))
```

Comment:
Can we align the error with [build.rs#L77-L81](https://github.com/hermit-os/kernel/blob/1f503cd8b8937cac564d495580bbab55d223065c/build.rs#L77-L81) and remove the output? It should get printed via the error message.

```suggestion
		Err(io::Error::new(
			io::ErrorKind::NotFound,
			"Could not find llvm-tools component\n\
			\n\
			Maybe the rustup component `llvm-tools` is missing? Install it through: `rustup component add llvm-tools`"
		))
```